prow/plugins/releasenote/releasenote.go

```diff
@@ -81,11 +84,12 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) err
 		return nil
 	}
 
-	// Only allow authors and assignees to add labels.
-	isAssignee := ic.Issue.IsAssignee(ic.Comment.User.Login)
+	// Only allow authors and org members to add labels.
+	isMember, _ := gc.IsMember(trustedOrg, ic.Comment.User.Login)
```

Comment:
I think it is functionally doing that now, the error does not give us any extra information. Anytime we get an error from IsMember we will get `(false, error)` which should fail the && check if the user is not the author.